### PR TITLE
add more load base split metrics (#10936)

### PR DESCRIPTION
**Pull request by ti-srebot**
cherry-pick #10936 to release-5.1
You can switch your code base to this Pull Request by using [git-extras](https://github.com/tj/git-extras):
```bash
# In tikv repo:
git pr https://github.com/tikv/tikv/pull/11627
```

After apply modifications, you can push your change to this PR via:
```bash
git push git@github.com:ti-srebot/tikv.git pr/11627:release-5.1-e6681ac0c0ee
```

---

Signed-off-by: lhy1024 <admin@liudos.us>

### What problem does this PR solve?

Issue Number: close #xxx 

Problem Summary:
add more load base split metrics

### What is changed and how it works?

Proposal: [xxx](url) 

What's Changed:

### Related changes

- Need to cherry-pick to the release branch

### Check List 

Tests 

- Unit test

### Release note 

```release-note
None.
```

### CODE REVIEW

**Comment by sticnarf** [comment]
@disksing @nolouch PTAL

File: metrics/grafana/tikv_details.json [hunk 1]
```diff
@@ -9098,6 +9098,5619 @@
           }
         },
         {
+<<<<<<< HEAD
```
Comment (nolouch) [inline metrics/grafana/tikv_details.json]: need resolve the conflict? cc @lhy1024

**Comment by sticnarf** [comment]
PTAL @nolouch @disksing

**Review by nolouch — APPROVED** [review]
lgtm

**Review by sticnarf — APPROVED** [review]

**Pull request merged**